lib/css_parser/parser.rb

```diff
@@ -277,7 +277,7 @@ def parse_block_into_rule_sets!(block, options = {}) # :nodoc:
             next
           end
 
-          if token =~ /\{/
```

Comment:
On line 287 of this file, we check to make sure the token isn't inside the string for "}", but not for "{". This means a malicious or malformed css could break the parser, as such:

```css
evil-div-one:before {
  content: "{";
} 
```